### PR TITLE
Safeframe ab Test

### PR DESCRIPTION
**Pull request by jeteve**
This implements https://trello.com/c/eY1QyceQ by adding a prebid safeframe AB test with one variant on 1% of the audience and changing the sonobi params accordingly.

### CODE REVIEW

File: common/app/conf/switches/ABTestSwitches.scala [hunk 1]
```diff
@@ -165,4 +165,15 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2019, 6, 5),
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-commercial-prebid-safeframe",
+    "Test the impact of serving prebid ads in safeframes",
+    owners = Seq(Owner.withGithub("jeteve")),
```
Comment (kelvin-chappell) [inline common/app/conf/switches/ABTestSwitches.scala]: Might want to give team ownership instead.

File: common/app/conf/switches/ABTestSwitches.scala [hunk 2]
```diff
@@ -165,4 +165,15 @@ trait ABTestSwitches {
...
+    safeState = Off,
+    sellByDate = new LocalDate(2019, 9, 30),
```
Comment (kelvin-chappell) [inline common/app/conf/switches/ABTestSwitches.scala]: What's happening next September?

**Review by kelvin-chappell — APPROVED** [review]
👍

**Comment by janua** [comment]
:+1:

**Pull request merged**